src/ActualLab.Rpc.Server/RpcWebSocketServer.cs

```diff
@@ -76,7 +81,10 @@ public virtual async Task Invoke(HttpContext context, bool isBackend)
             }
         }
         finally {
-            webSocket?.Dispose();
+            if (channel is not null)
```

Comment:
There is no need to dispose the channel, coz at worst it will dispose the websocket. Dispositing the websocket guarantees the channel will die too.